exercises/concept/need-for-speed/.docs/introduction.md

```diff
@@ -1,18 +1,46 @@
 # Introduction
 
-In Go, a `struct` is a sequence of named elements called _fields_, each field having a name and type. The name of a field must be unique within the struct. `Structs` can be compared with the _class_ in the Object Oriented Programming paradigm.
+In Go, a `struct` is a sequence of named elements called _fields_, each field has a name and type. The name of a field must be unique within the struct.
+`Structs` can be compared with _classes_ in the Object-Oriented Programming paradigm.
 
-You create a new struct by using the `struct` keyword, a **_built-in type_**, and explicitly define the name and type of the fields as shown in the example below.
+You create a new struct by using the **_built-in type_** keywords `type` and `struct`, and explicitly define the name and type of the fields.
```

Comment:
This now reads "using the built-in type keywords type ...". I think it should say "Using the `type` keyword and the built-in type `struct` ..."